src/components/ItemList.js

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import { View, Text, Image } from 'react-native';
+import styles from './ItemList.styles';
+import Button from './Button';
+import activeCheckbox from '../assets/icons/iconCheckboxActive.png';
+import inactiveCheckbox from '../assets/icons/iconCheckboxInactive.png';
+
+const ItemList = (props) => {
+  const {
+    item:
+    {
+      title, description, done, id,
+    }, onClickAction,
+  } = props;
+  const {
+    container, textTitle, textDescription, text, checkBox, button,
+  } = styles;
+
+  this.renderButton = () => {
```

Comment:
I think it's better to put this function outside `ItemList`. You're kinda using global variable scoping here (limited to the scope of `ItemList`, granted). 

I personally prefer defining explicit functions that receive all necessary parameters.